server/controllers/BookController.js

```diff
@@ -75,6 +75,65 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* Edit Books
+* @name editBook
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+* @memberof BookController
+*/
+  static async editBook(request, response) {
+    const { id } = request.params;
+    const {
+      title, description, isbn, price, yearPublished, authorId
+    } = request.body;
+    try {
+      const books = await Book.findAll();
+      const availableBook = books.find(book => book.id === +id);
+      if (!availableBook) return responseMessage(response, 400, { message: 'book does not exist' });
+      if (isbn) {
+        const availableIsbn = books.map(book => book.isbn).find(data => data === isbn);
+        if (availableIsbn === isbn) return responseMessage(response, 409, { message: 'isbn already exist' });
+      }
+      const results = await Book.update({
+        title, description, isbn, price, yearPublished
```

Comment:
Assuming the admin only inputs the title in the request body, and the title is the exact same thing as what was previously there, from your logic in this controller, the response would be 'book successfully updated'. Should that be the correct response, even though there was no update?

Reply:
once you have requested for it, it is marked as updated